addons/calendar/wizard/calendar_provider_config.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.addons.base.models.ir_module import assert_log_admin_access
+
+
+class CalendarProviderConfig(models.TransientModel):
+    _name = 'calendar.calendar.provider.config'
```

Comment:
Why two times calendar word in model name ?

Reply:
Why is originally this, as you asked, but I'll change as you're puzzled.

In `calendar`, a "Calendar Provider Configuration Wizard"

If it was a "Pancake Recipe Configuration", I would have called it `calendar.pancake.recipe.config`